Jenkinsfile.trigger

```diff
@@ -161,12 +181,47 @@ for (buildObj in queue) {
 				quietPeriod: 5, // seconds
 			)
 			// TODO do something useful with "res.result" (especially "res.result != 'SUCCESS'")
-			// (maybe store "res.startTimeInMillis + res.duration" as endTime so we can implement some amount of backoff somehow?)
 			echo(res.result)
 			if (res.result != 'SUCCESS') {
+				def c = 1
+				if (pastFailedJobs[buildObj.buildId]) {
+					c += pastFailedJobs[buildObj.buildId].count
```

Comment:
Something to think about in the future / next round: if `count` isn't set for some reason (but the object exists), this will be `null` and we'll get an execption from `java.lang.Integer#plus` (_shouldn't_ matter for this because we're the only ones creating this file and it should _always_ be well-formed, but this will be a good reminder for if/when something goes wrong that this line might need to be more defensive against the inputs).